### DISCUSSION TITLE
!NEW! Guest sign-in NO !SH1mmer needed!

### DISCUSSION THREAD

**Discussion by Thecodeisrereall** [3kh0/ext-remover#1113] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I am currently writing this from the guest tab RN. I'm on chrome version 125 i just stumbled across this while trying to continue with another exploit I found. This requires you to power wash your device then at the WiFi-screen to login to a network where you will not have access to the internet. MAC address is blocked or something like that. Then it will show something of hey you cant access this try **using guest browsing**. And use that.
 

Google Chrome	125.0.6422.134 (Official Build) (64-bit) 
Revision	538cce6c6ced290f7b1093a2b1c92303595e4b8f-refs/branch-heads/6422@{#1134}
Platform	15853.47.0 (Official Build) stable-channel dedede
Firmware Version	Google_Cret.13606.601.0
Customization ID	cret

**Comment by akabutnicer** [comment]
If you could, possibly go further into depth for those who aren't educated in this topic.
